public/stylesheets/account.css

```diff
@@ -105,3 +117,67 @@ main {
   margin-top: -3px;
   font-size: 9px;
 }
+
+@media (max-width: 1024px) {
+  body {
+    font-size: 10px;
+  }
+  .profile-img {
+    width: 95px;
+    height: 95px;
+  }
+  .sidebar-tab {
+    font-size: 12px;
+    padding: 10px 10px;
+  }
+  .account-first-row {
+    min-height: 131px;
+  }
+  main {
+    padding: 70px 8% 70px 5%;
+  }
+}
+
+@media (max-width: 576px) {
+  .profile-img {
+    width: 130px;
+    height: 130px;
+  }
+  .main-box {
+    display: block;
+  }
+  .sidebar,
+  .info-box {
+    width: 100%;
+  }
+  .sidebar {
+    border-right: none;
+    margin-bottom: 50px;
```

Comment:
This margin looks rather odd: 
![image](https://github.com/DmitrijSidorkin/pixel-thing/assets/9388169/58ef4add-ca0f-4854-a1cb-d597d699cb46)